tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
It looks like, this file is not prettified.
This would make it difficult to identify upcoming changes if the file is formatted in the same change.